qubes/securedrop.Proxy

```diff
@@ -1 +1 @@
-/usr/bin/sd-proxy /etc/sd-proxy.yaml
+/usr/bin/sd-proxy /home/user/.securedrop_proxy/sd-proxy.yaml
```

Comment:
If I understand correctly, this path will only exist after running the template consolidation logic [1]. if we do not configure both the existing path and the new path here, the proxy will only work in either the non-consolidated or the consolidated template approach. In other words, this proxy change will break existing installs, until the dom0 logic is run again.

https://github.com/freedomofpress/securedrop-workstation/pull/619/files#diff-6665f59c22d94742e30d7bbf295194ceR7

Reply:
>  In other words, this proxy change will break existing installs, until the dom0 logic is run again.

A critical assumption of the template consolidation plan in https://github.com/freedomofpress/securedrop-workstation/pull/619 is that the SecureDrop Workstation components are updated _only via the SDW GUI updater_. As long as we accept that limitation, then the new updater logic will ensure that the config file is handled appropriately.